extensions/resteasy-reactive/rest-client-reactive-jackson/runtime/src/main/java/io/quarkus/rest/client/reactive/jackson/runtime/serialisers/ClientJacksonMessageBodyWriter.java

```diff
@@ -15,7 +15,7 @@
 import jakarta.ws.rs.WebApplicationException;
 import jakarta.ws.rs.core.MediaType;
 import jakarta.ws.rs.core.MultivaluedMap;
-import jakarta.ws.rs.ext.MessageBodyWriter;
+import `jakarta.ws.rs.ext.MessageBodyWriter`;
```

Comment:
What is the motivation for this change?

Reply:
I think it won't compile...

Reply:
I have no idea how that happened. It should be fixed now